src/components/shared/Col.js

```diff
@@ -0,0 +1,34 @@
+import { useResponsive } from 'hooks/useResponsive'
+import { Box } from 'grommet'
+import styled, { css } from 'styled-components'
+
+const Wrapper = styled(Box)`
+  ${({ flexValue }) => css`
+    flex: ${flexValue};
```

Comment:
Couple things here, I think the name `ColBox` makes more sense here. Also, there are attributes you can use to manage the flex type.

Reply:
Previously we talked about this naming and instead of using the short name, we renamed it as [`Column`](https://github.com/AlexsLemonade/refinebio-web/blob/c06a2081f4b498784436e625a09cb07726dee81f/src/components/shared/Column.js) (for PR https://github.com/AlexsLemonade/refinebio-web/pull/32 and up). 

For the flex type, I need to set the [flex-basis](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) and couldn't find the prop for it in the [Grommet Box](https://v2.grommet.io/box#flex). Let me know, thank you!

Reply:
https://v2.grommet.io/box#basis

Reply:
This change will be applied to PR [29 - Homepage (Without Cached Data)](https://github.com/AlexsLemonade/refinebio-web/pull/32) and up, since this component doesn't exist any more, thank you!